src/main/webapp/app/shared/orion/orion-connector.service.ts

```diff
@@ -130,7 +130,7 @@ export class OrionConnectorService implements ArtemisOrionConnector {
      *
      * @param buildErrors All compile errors for the current build
      */
-    onBuildFailed(buildErrors: BuildLogErrors) {
+    onBuildFailed(buildErrors: Array<Annotation>) {
```

Comment:
does this type change introduce errors in the orion plugin? I am not sure if it can handle that correctly

Reply:
i will look into it

Reply:
I added code to transform the annotations back to the _old_ format used by orion in [df90358](https://github.com/ls1intum/Artemis/pull/2012/commits/df903583823d35fbb892fbbbdf58ce1dae53bc67)